README.md

```diff
@@ -40,13 +42,58 @@ See the generated `README.md` file for more details.
 
 ### Test Project Pack
 
-You can open `test-project-pack/test-project-pack.cogs` in COGS, which automatically includes your plugins and/or custom Media Master content. 
+You can open `test-project-pack/test-project-pack.cogs` in COGS, which automatically includes your plugins and/or custom Media Master content.
 In development mode (`yarn start`) they will live-reload with any changes you make.
 In production mode (`yarn build`) the production versions of your plugins/custom Media Master content will included.
 
 > [!NOTE]
 > The template is configured to use Yarn 4 by default. It should be possible to use other package managers such as `npm` or `pnpm`, but you will need to manually configure these after you have created a project from the example template.
 
+## COGS Media Master Custom Content
+
+If you have created Custom Content:
+
+- Go to the Connect tab.
+- Add a Media Master
+- In the Media Setup panel, set Media type to Custom
```

Comment:
Shall we just add this to the test project pack so you don't have to do it every time?